src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -278,138 +275,137 @@ public final void apply(Project project) {
+                .collect(Collectors.toMap(Functions.identity(), subproject -> {
+                    VersionsLockExtension ext = subproject.getExtensions().getByType(VersionsLockExtension.class);
+                    LockedConfigurations lockedConfigurations = computeConfigurationsToLock(subproject, ext);
+                    addConfigurationDependencies(
+                            subproject,
+                            subproject.getConfigurations().getByName(CONSISTENT_VERSIONS_PRODUCTION),
+                            lockedConfigurations.productionConfigurations());
+                    addConfigurationDependencies(
+                            subproject,
+                            subproject.getConfigurations().getByName(CONSISTENT_VERSIONS_TEST),
+                            lockedConfigurations.testConfigurations());
+                    return lockedConfigurations;
+                }));
     }
 
     /**
-     * This method sets up the necessary intermediate configurations in each project, and wires up the
-     * dependencies from {@link #UNIFIED_CLASSPATH_CONFIGURATION_NAME} to these configurations.
-     * It doesn't wire up the actual configurations that we intend to lock, because that will be done later, in
-     * afterEvaluate.
+     * This method sets up the necessary intermediate configurations in each project, and wires up the dependencies from
+     * {@link #UNIFIED_CLASSPATH_CONFIGURATION_NAME} to these configurations. It doesn't wire up the actual
+     * configurations that we intend to lock, because that will be done later, in afterEvaluate.
      */
-    private void setupDependenciesToProject(
-            Project rootProject,
-            Configuration unifiedClasspath,
-            Project project) {
+    private void setupDependenciesToProject(Project rootProject, Configuration unifiedClasspath, Project project) {
         // Parallel 'resolveConfigurations' sometimes breaks unless we force the root one to run first.
         if (rootProject != project) {
-            project.getPluginManager().withPlugin("com.palantir.configuration-resolver", plugin -> {
-                project.getTasks().named("resolveConfigurations", task -> task.mustRunAfter(":resolveConfigurations"));
-            });
+            project.getPluginManager()
+                    .withPlugin("com.palantir.configuration-resolver", plugin -> {
+                        project.getTasks()
+                                .named("resolveConfigurations", task -> task.mustRunAfter(":resolveConfigurations"));
+                    });
         }
 
         // This is not how we collect dependencies, but is only meant to capture and neutralize the user's
         // inter-project dependencies.
-        project.getConfigurations().register(PLACEHOLDER_CONFIGURATION_NAME, conf -> {
-            conf.setVisible(false).setCanBeResolved(false);
+        project.getConfigurations()
+                .register(PLACEHOLDER_CONFIGURATION_NAME, conf -> {
```

Comment:
Interesting that a lot of the diff in this PR is because we seem to frequently have the pattern:

```java
foo.something().other().method("arg1", arg2, lambda -> {
  // lots of code
}
```

Reply:
I think that works in the latest version.
For this code however `project.getConfigurations()` has been pulled onto the `=` line: https://github.com/palantir/gradle-consistent-versions/pull/305/files#diff-56e0cf074cb9bc301934d0fd45ed3508R56-R59